src/api/__tests__/api.spec.js

```diff
@@ -245,6 +245,18 @@ describe('Api', () => {
                 .catch(done);
         });
 
+        it.only('should reject with an error when url contains encoded query string', (done) => {
+            const message = 'Cannot process URL encoded URLs, pass an unencoded URL';
+
+            api.get('test?one=%5Bwith%20a%20filter%5D')
+                .then(() => done(new Error('The request should error')))
```

Comment:
Consider removing the code inside of `done`, as it is not needed and possibly obscures what is happening in the test. you could also use process.nextTick:

```process.nextTick(() => {
                api.get('test?one=%5Bwith%20a%20filter%5D')
                    .catch((err) => {
                        expect(err).toBeInstanceOf(Error);
                        expect(err.message).toBe(message);
                        done();
                    });
            });```